examples/rbac/README.md

```diff
@@ -131,20 +130,24 @@ The Deployment of the overall setup looks like :
 
 #### **Validating Permissions examples.**
 
-As we created two examples under the client folder: `test_admin.py` and `test_user.py`. Both examples aim to perform actions related to Historical Features, Materialize Incremental, and Online Features.
-According to the roles:
+* As the the  [test.py](client/feature_repo/test.py) here test example try to fetch Historical Features, do Materialize Incremental, and fetch Online Features and push to online/offline based on user roles.
```

Comment:
some typos in this sentence `As the the test.py here test example try to fetch ...`

Reply:
Fix that